Common/Data/Custom/TradingEconomics/TradingEconomicsCalendar.cs

```diff
@@ -242,6 +335,37 @@ public override string ToString()
             return Invariant($"{symbol} ({Country} - {Category}): {Event} : Importance.{Importance}");
         }
 
+        /// <summary>
+        /// Convert this instance to a CSV file (tab delimited)
+        /// </summary>
+        /// <returns>string as CSV</returns>
+        public string ToCsv()
+        {
+            return string.Join(",",
+                EndTime.ToStringInvariant("yyyyMMdd HH:mm:ss"),
+                LastUpdate.ToStringInvariant("yyyyMMdd HH:mm:ss"),
+                Ticker,
+                Actual.ToStringInvariant(),
+                CalendarId,
+                Category,
+                Country,
+                (int)DataType,
+                DateSpan,
+                $"\"{EventRaw}\"",
```

Comment:
Could we skip re-adding the double quotes here? Same bellow

Reply:
These fields can contain the delimiter in their value (`,`), which we escape by placing the entire value around double quotes. Unfortunately, they can not be removed